test/blackbox-tests/test-cases/env/env-var-expansion.t/run.t

```diff
@@ -32,7 +32,6 @@ incrementality works properly, that (setenv ...) is taken into account, etc.
   Entering directory 'correct'
   $ DUNE_ENV_VAR=true dune build --root correct @echo1
   Entering directory 'correct'
-  true
```

Comment:
I was initially confused by this, but in fact we don't re-run the action simply because we memoise such actions for longer.